final_project_schema.sql

```diff
@@ -0,0 +1,190 @@
+CREATE DATABASE finalProject; --run me first!
+
+DROP DATABASE finalProject; --run me if you mess up
+
+DROP TABLE IF EXISTS users; --run the rest of the code now
+
+CREATE TABLE IF NOT EXISTS users (
+    username VARCHAR(50),
+    password VARCHAR(50) NOT NULL, --may need to change the length once we decide hashing length, for now store in plaintext
+    first_name VARCHAR(20) NOT NULL,
+    last_name VARCHAR(20) NOT NULL,
+    PRIMARY KEY (username)
+);
+
+DROP TABLE IF EXISTS vehicle;
+
+CREATE TABLE IF NOT EXISTS vehicle (
+    vehicle_id UUID,    -- this is just a basic vehicle
+    make VARCHAR(50) NOT NULL,
+    model VARCHAR(50) NOT NULL,
+    year CHAR(4) NOT NULL,
+    PRIMARY KEY (vehicle_id)
+);
```

Comment:
There should be a foreign key for a users username. Each car has one owner.

Reply:
Multiple people can have similar make, models and years, I think its important to add the username in this table instead of creating a separate table

Reply:
I can revise it to delete the user_vehicle table and add a foreign key to this one.